spec/error_creator_spec.rb

```diff
@@ -124,6 +124,28 @@ module ExceptionHunter
               end
             end
           end
+
+          context 'when async logging is set to true' do
+            let!(:original_queue_adapter) { ActiveJob::Base.queue_adapter }
+            let(:async_logging)           { true }
+            let(:tag)                     { nil }
+
+            before do
+              ActiveJob::Base.queue_adapter = :test
+
+              allow(ExceptionHunter::Config)
+                .to receive(:async_logging)
+                .and_return(async_logging)
+            end
+
+            after do
+              ActiveJob::Base.queue_adapter = original_queue_adapter
```

Comment:
Do we even need to change it after? I'm not against it, just wondering if there was any reason

Reply:
it was not necessary, I removed it